src/H5Oprivate.h

```diff
@@ -684,7 +684,7 @@ typedef struct H5O_bogus_t {
  */
 typedef struct H5O_ginfo_t {
     /* "Old" format group info (not stored) */
-    uint32_t lheap_size_hint; /* Local heap size hint              */
+    size_t lheap_size_hint; /* Local heap size hint              */
 
```

Comment:
I probably need to go look at the file format to make sure this value makes sense.

Reply:
Yes, the file format specifies this as a 32-bit value, so allowing it to be a size_t is probably not a good direction.

Reply:
Right. I'm switching it back.